app/src/main/java/com/practicum/playlistmaker/media/ui/MediaFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.practicum.playlistmaker.media.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayoutMediator
+import com.practicum.playlistmaker.R
+import com.practicum.playlistmaker.databinding.FragmentMediaBinding
+
+class MediaFragment: Fragment() {
+    private var _binding: FragmentMediaBinding? = null
```

Comment:
🍏 Можно использовать lateinit var, чтобы не инициализировать переменную с помощью null.
```suggestion
    private lateinit var binding: FragmentMediaBinding
```

https://kotlinlang.org/docs/properties.html